src/ShadowDOM/wrappers/Selection.js

```diff
@@ -56,6 +53,15 @@
     }
   };
 
+  // Not all browsers support Selection.extend. Some versions of IE do not support extend, code that checks
+  // if extend exists in the Selection would fail the test if we define extend on the wrapper and it does not exist in
+  // the browser Selection object.
+  if (window.getSelection().extend){
```

Comment:
also, I think you could do: `if (OriginalSelection.prototype.extend) {` and avoid calling getSelection